docs/proposals/prop-2-releases.md

```diff
@@ -0,0 +1,72 @@
+Proposal: 1
+Title: Releasing Anonlink
+Author: Brian Thorne
+Status: Draft
+Created: 2019-11-5
+
+## Abstract
+
+This proposal outlines a design for automated packaging and releasing of anonlink using our primary CI/CD system - 
+Microsoft's Azure DevOps [^1].
+
+## Motivation 
+
+An automated process for releasing Anonlink allows for more frequent releases and ensures that releases are
+packaged, reviewed and communicated consistently.
+
+### Current Manual Process
+
+The process that team members carry out before adoption of this proposal is as follows:
+
+ - Update the version in `setup.py`. Use semantic versions.
+ - Update the `CHANGELOG.rst`
+ - Create a `release-{version}` branch from an up to date master branch and request a code review.
+ - Tag the commit with the version by creating a GitHub release https://github.com/data61/anonlink/releases, include the changelog.
+ - This triggers the CI systems (Travis and Azure) to build and upload artifacts to PyPi.
+ - check on [pypi](https://pypi.org/project/anonlink/#history) that release appears and then check that it is indeed pip installable.
+ - Manually upload release artifacts (such as binary wheels and the tar.gz file) to the GitHub release page.
+ - proudly announce the new release on the anonlink google group https://groups.google.com/forum/#!forum/anonlink
+
```

Comment:
Erm we only have internal documentation at this point? anonlink's docs don't really exist yet unfortunately.

I agree that if accepted, the documentation and implementation will be the source of truth rather than the proposal